app/components/windows/AddSource.vue.ts

```diff
@@ -114,6 +115,7 @@ export default class AddSource extends Vue {
             propertiesManagerSettings: this.sourceAddOptions.propertiesManagerSettings
           }
         );
+        
```

Comment:
Please don't commit a file if it has only formatting changes and you base task is not something like refactoring or reformatting.
Motivation:
I want to always be able to check what files have been affected by the commit in git history. Files changes like this give me disinformation.